### PR TITLE
chore(changelog): Adds lifecycle breaking change to the changelog

### PR DESCRIPTION
**Pull request by ericmdantas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,21 +34,64 @@
...
+import {Component, View, LifecycleEvent} from 'angular2/angular2';
+
+@Component({
+  selector: 'hello-cmp',
+  lifecycle: [LifecycleEvent.onInit]
+})
+@View({
+  template: `<h1>hello, there!</h1>`
+})
+
```
Comment (vicb) [inline CHANGELOG.md]: nit: no blank line here

Could you please also add an example for ES5, see the the [original commit message](https://github.com/vicb/angular/commit/ecfed76847beb7450b88aa0a7c0f1e7d62a638e4)

**Comment by ericmdantas** [comment]
@vicb done.

**Comment by vicb** [comment]
LGTM, thanks

**Comment by btford** [comment]
@vicb can you please make sure the commit messages match our format next time before merging? Thanks.

**Pull request closed without merging**